packages/esm-patient-conditions-app/src/conditions/conditions-detailed-summary.component.tsx

```diff
@@ -92,11 +92,11 @@ function ConditionsDetailedSummary({ patient }) {
             <div className={styles.filterContainer}>
               <Dropdown
                 id="conditionStatusFilter"
-                initialSelectedItem={t('active', 'Active')}
+                initialSelectedItem="Active"
```

Comment:
Doesn't this undo our ability to translate these strings?

Reply:
Ah... yeah, I see. So, theoretically, there would be a way for the backend to localise the display value, but that's not what's being operated on here.

In the [FHIR Condition](https://www.hl7.org/fhir/condition.html), `clinicalStatus` takes a concept from a required [ValueSet](https://www.hl7.org/fhir/valueset-condition-clinical.html), so, e.g., "active" is not intended to be a display string "active"; it's actually the concept ID for the "active" concept that might have different translations in different locales. From that perspective, [this](https://github.com/openmrs/openmrs-esm-patient-chart/blob/f12632e57e02fb77fc612778937bc3ad0a6dd04a/packages/esm-patient-conditions-app/src/conditions/conditions.resource.tsx#L91) is a bit odd.

Let me think about how we might translate this best... It would be nice if we didn't end up in a situation where we needed to synchronize both backend and frontend translation strings.